modules/grabbag/grabbag.js

```diff
@@ -1,10 +1,9 @@
 !(function(){
 
   //check on load to see if you need to be dragged back after wrecking
-  if (location.href.indexOf('grabbag')){
+  if (location.pathname.indexOf('grabbag') >= 0){
```

Comment:
Testing on Chrome, `location.pathname` omits the `#grabbag` entirely, whereas `location.href` preserves it.  Since grabbag is just an identifier, this change breaks the test.